pangocairo/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
+#![allow(clippy::should_implement_trait)]
```

Comment:
Also here

Reply:
Ah too late. I put the attribute on the methods directly.